docker-compose.yml

```diff
@@ -22,12 +22,15 @@ services:
 
   web:
     build: .
+    env_file:
+      - env.env
     tty: true
     command: ./init.sh
```

Comment:
Should this `init.sh` command remain here, if it is added to the Dockerfile? Wouldn't that result in having 2 instances of the server running at the same time?

Reply:
No, since I put the init script as a command in the Dockerfile the docer-compose file overrides the Dockerfile command.  It is a bit redundant, so I'll take it out though.

Reply:
Fixed in latest commit https://github.com/chibicitiberiu/ytsm/pull/21/commits/1773f83a3c4e1a538dfb4f444d2a1b267dc00ad2

Reply:
Great, thanks 👍